plugins/catalog-backend/api-report.md

```diff
@@ -69,11 +71,7 @@ export type AnalyzeLocationRequest = AnalyzeLocationRequest_2;
 // @public @deprecated (undocumented)
 export type AnalyzeLocationResponse = AnalyzeLocationResponse_2;
 
-// @public (undocumented)
-export type AnalyzeOptions = {
-  url: string;
-  catalogFilename?: string;
-};
+export { AnalyzeOptions };
```

Comment:
these two should instead be added to `src/depracated.ts` (check that file for the patterns used) and marked as such